packages/veui/test/unit/specs/plugins/alert.spec.js

```diff
@@ -0,0 +1,15 @@
+import { createLocalVue } from '@vue/test-utils'
+import alertPlugin from '@/plugins/alert'
+
+const localVue = createLocalVue()
+
+localVue.use(alertPlugin)
+
+describe('plugins/alert', () => {
+  it('should add `alert` methods to Vue prototype', () => {
```

Comment:
```suggestion
  it('should add `alert` methods to Vue prototype', () => {
    expect(localVue.prototype.$alert).to.be.a('function')
```

`$alert` 本身也是个 function